doc/source/installation.rst

```diff
@@ -0,0 +1,172 @@
+Installation
+============
+
+SIFT is provided either as Conda packages or as a compressed all-in-one
+bundle (tarball/zip) of all dependencies with a single start script.
+Unless otherwise documented below, it is recommended that the software
+bundle be used for any novice users as it requires the least amount of
+prior execution of conda or command line tools. Conda-based installation
+is best for testers, developers, and those wishing to customize versions
+of dependencies used by SIFT.
+
+System Requirements
+-------------------
+
+SIFT works on Windows, Mac, and Linux, but depends on many complex libraries
+and relies heavily on GPU (video card) processing via the PyOpenGL and VisPy
+libraries which may make installation difficult on some platforms or systems.
+In general it is best to have a local physical system to run SIFT as opposed
+to a virtual machine or remote server. Virtual machines and remote servers
+commonly have limited ways of displaying OpenGL visualizations to another
+machine.
```

Comment:
I would say "relies heavily on GPU" is a bit too strong of a statement, no? It seems to me that it works ok also on simpler systems without it. I'm just afraid that this may scare off potential users that have limited resources. 
```suggestion
SIFT works on Windows, Mac, and Linux, but depends on many complex
libraries which may make installation difficult on some platforms or systems.
It also makes heavy use of GPU (video card) processing via the PyOpenGL and VisPy
libraries. In general it is best to have a local physical system to run SIFT as opposed
to a virtual machine or remote server. Virtual machines and remote servers
commonly have limited ways of displaying OpenGL visualizations to another
machine.
```